CHARTER.md

```diff
@@ -0,0 +1,104 @@
+# MapLibre Charter
+
+## Definitions
+
+*MapLibre Organization:* An umbrella organization to develop, maintain and promote MapLibre software projects
+
+*MapLibre Projects:* multiple repositories, each being some part of the map visualization stack from the raw data to the map visualization and interaction.
+
+*MapLibre:* As a single word, “MapLibre” is a brand name used by the front-end map rendering MapLibre projects (i.e. code libraries).
```

Comment:
I'd have expected this to be the description for "MapLibre GL". There are also other MapLibre Projects which might not be "front-end map rendering".

Overall, looking over the project list, there seems to be no clear consensus on how projects should be named (yet).

Reply:
nit: The headline of this document also refers to the "MapLibre"-Charter, which might be misleading as only affecting those projects.

Reply:
The intention here is to preserve the MapLibre "brand" for rendering libraries, so even if other projects under the organizational umbrella eventually offer alternatives to our sponsors' own projects, there's no conflict for them to promote MapLibre by name.

I think there's room for a healthy discussion on the pros/cons of this and how it affects our project naming approach.

Reply:
Using the name "MapLibre" for frontend rendering projects goes back to the discussion about the scope of the MapLibre Organization, see https://github.com/maplibre/maplibre/discussions/27. Practically what it means for us is that we want to put the name "MapLibre" into project names which have to do with frontend rendering.

Examples of good project names:
* :heavy_check_mark: maplibre-gl-js 
* :heavy_check_mark: maplibre-gl-native
* :heavy_check_mark: maplibre-rs
* :heavy_check_mark: maplibre-gl-directions

Examples of what would be bad project names:
* :heavy_multiplication_x: maplibre-tileserver
* :heavy_multiplication_x: maplibre-tile-generator

Reply:
Feel free to reopen this conversation @JannikGM. I will close it for now...